gloss-examples/raster/Crystal/Main.hs

```diff
@@ -89,13 +91,13 @@ parseArgs args config
         | otherwise
         = do    printUsage
                 exitWith $ ExitFailure 1
-
+  where (screensizeX,screensizeY) = unsafePerformIO screensize
```

Comment:
We can't require unsafePerformIO in the client code. These examples are supposed to demonstrate to beginners how to write nice Haskell programs using Gloss. Can we instead perform some standard IO action in the main function at startup time?